### PR TITLE
chore: remove get-uri and hasbin

### PR DESCRIPTION
**Pull request by Kirill89**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Remove dependencies:

- `get-uri`
- `hasbin`

Because nobody use it in the code.

### CODE REVIEW

**Comment by michael-go** [comment]
@Kirill89 if you're at it - did you try running https://www.npmjs.com/package/depcheck ?

**Comment by Kirill89** [comment]
@michael-go.

```
> depcheck
Unused dependencies
* get-uri
Unused devDependencies
* @types/diff
* @types/lodash
* @types/node
```

**Comment by michael-go** [comment]
interesting it didn't flag `hasbin`

**Comment by Kirill89** [comment]
I bet it's because you can find it in [assets](https://github.com/snyk/snyk/blob/68d5c6d0ed8efe1c9124ce9834973e57fe79ec69/test/fixtures/protect-via-snyk/yarn.lock#L339). But it's not related to code.

File: package.json [hunk 1]
```diff
@@ -57,8 +57,6 @@
     "configstore": "^3.1.2",
     "debug": "^3.1.0",
     "diff": "^4.0.1",
-    "get-uri": "2.0.2",
-    "hasbin": "^1.2.3",
```
Comment (adrukh) [inline package.json]: Maybe worth relocking `test/fixtures/protect-via-snyk` to get rid of `hasbin` there as well?

Comment (Kirill89) [reply package.json]: We don't have lock file here.

Comment (Kirill89) [reply package.json]: Does it make sense? It's only test fixtures.

Comment (adrukh) [reply package.json]: I'm 90% sure that without this change, running `npm t` locally will generate uncommitted changes on the repo. OTOH, we'd need to re-lock the test project only after this change is released... Just a nag, not a must :)

**Pull request merged**